### ISSUE TITLE
Entity value contains units of measurement, should be an attribute

### ISSUE THREAD

**Issue by Plawasan** [craibo/ha_strava#38]
Hi,

I'd like to use the summary entities for weekly/monthly stats however currently their state includes not only the numeric value but also the unit making the entity state a string that cannot be directly used in e.g. utility meters:

```
{{ states("sensor.strava_stats_summary_all_run_distance") }}

Result type: string
245.54 km
```

If I'm not mistaken the correct way to handle this is to move the "km" in this case to an unit_of_measurement attribute.

Thanks!

**Comment by craibo** [comment]
Thanks for raising @Plawasan 

I can do that, it is already on my list of things to update. 
I'll push it up the list

**Comment by craibo** [comment]
This has been corrected in 2.1.1 release
